README.md

```diff
@@ -32,7 +33,6 @@ data/
 ├─ model/
 ├─ raw/
 ├─ wrangle/
```

Comment:
would it be useful to re-order these in terms of the actual order of the workflow i.e. raw -> wrangle -> model ?